apps/rush-lib/src/api/VersionPolicyConfiguration.ts

```diff
@@ -24,13 +25,15 @@ export interface ILockStepVersionJson extends IVersionPolicyJson {
   version: string;
   nextBump: string;
   mainProject?: string;
+  includeEmailInChange?: boolean;
 }
 
 /**
  * @beta
  */
 export interface IIndividualVersionJson extends IVersionPolicyJson {
   lockedMajor?: number;
+  includeEmailInChange?: boolean;
```

Comment:
Note: these interfaces extend from `IVersionPolicyJson`, so you don't need to specify them again (they'll be inherited from the parent interface).